src/vcpkg/build.cpp

```diff
@@ -107,6 +107,16 @@ namespace
                                  "Printed after the name of an installed entity to indicate that it was successfully "
                                  "downloaded but no build or install was requested.",
                                  "DOWNLOADED");
+
+    DECLARE_AND_REGISTER_MESSAGE(BuildingPackageFailed,
+                                 (msg::spec, msg::build_result),
+                                 "",
+                                 "building package {spec} failed with: {build_result}");
```

Comment:
```suggestion
                                 "building {spec} failed with: {build_result}");
```

As an aside, I think "package foo" can always be better replaced with just "foo". I know this is a change from the status quo so feel free to say "that's another PR Bill"